src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -138,21 +238,38 @@ private IEnumerable<string> GetWatchedRules(RuleHandlerType handlerType)
                            .ToArray();
         }
 
+        private bool HasTargetFrameworksChanged(IProjectVersionedValue<IProjectSubscriptionUpdate> e, out string targetFrameworks)
+        {
+            IProjectChangeDescription projectChange;
+            if (e.Value.ProjectChanges.TryGetValue(ConfigurationGeneral.SchemaName, out projectChange) &&
```

Comment:
I did remember hitting this case where it didn't have this key, and switched to TryGetValue. I'll try to switch back and see if I can identify the scenario.